101-application-gateway-rewrite/azuredeploy.json

```diff
@@ -0,0 +1,380 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "addressPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.0.0/16",
+      "metadata": {
+        "description": "Address prefix for the Virtual Network"
+      }
+    },
+    "subnetPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.0.0/28",
+      "metadata": {
+        "description": "Subnet prefix"
+      }
+    },
+    "skuName": {
+      "type": "string",
+      "defaultValue": "standard_v2",
+      "metadata": {
+        "description": "Sku Name"
+      }
+    },
+    "backendIpAddress1": {
+      "type": "string",
+      "defaultValue": "10.0.1.10",
+      "metadata": {
+        "description": "IP Address for Backend Server 1"
+      }
+    },
+    "backendIpAddress2": {
+      "type": "string",
+      "defaultValue": "10.0.1.11",
+      "metadata": {
+        "description": "IP Address for Backend Server 2"
+      }
+    },
+    "MSIVersion": {
```

Comment:
remove this parameter, apiVersions need to be literal values

Reply:
Removed.